### PR TITLE
Use limited array printing for OneHotArrays

### LINKED ISSUE #1603: Bad printing for OneHot*
MWE: Note that this prints out the entire array.

```julia
julia> l = 1:1000
1:1000

julia> Flux.onehotbatch(l[1:2], l)
1000×2 Flux.OneHotArray{1000,2,Vector{UInt32}}:
 1  0
 0  1
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
[...]
```

### PR DESCRIPTION
**Pull request by darsnack**
Fix #1603. May affect doctests so we might need an extra commit to fix those.

### PR Checklist

- ~~[ ] Tests are added~~
- ~~[ ] Entry in NEWS.md~~
- ~~[ ] Documentation, if applicable~~
- ~~[ ] API changes require approval from a committer (different from the author, if applicable)~~

### CODE REVIEW

**Comment by darsnack** [comment]
Looks like this now:
```julia
julia> l = 1:1000
1:1000

julia> Flux.onehotbatch(l[1:2], l)
1000×2 Flux.OneHotArray{1000,2,Vector{UInt32}}:
 1  0
 0  1
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 ⋮  
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
 0  0
```

File: src/onehot.jl [hunk 1]
```diff
@@ -20,7 +20,7 @@ OneHotMatrix(indices, L) = OneHotArray(indices, L)
 
 function _show_elements(x::OneHotArray)
   xbool = convert(Array{Bool}, cpu(x))
-  xrepr = join(split(repr(MIME("text/plain"), xbool), "\n")[2:end], "\n")
+  xrepr = join(split(repr(MIME("text/plain"), xbool; context=:limit => true), "\n")[2:end], "\n")
```
Comment (DhairyaLGandhi) [inline src/onehot.jl:23]:
```suggestion
  xrepr = join(split(repr(MIME("text/plain"), xbool; context= :limit => true), "\n")[2:end], "\n")
```

**Review by DhairyaLGandhi — APPROVED** [review]
Thanks

**Review by DhairyaLGandhi — APPROVED** [review]
bors r+

**Pull request merged**